build.gradle

```diff
@@ -459,9 +469,7 @@ installMacDist {
     }
 }
 
-def hunspellJar = configurations.runtimeClasspath.fileCollection {
```

Comment:
runtimeClasspath can be dictionary [group, name, version], or set of File by fileTree()
When created y fileTree(), it does not have name.